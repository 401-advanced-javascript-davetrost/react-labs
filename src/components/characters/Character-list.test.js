import React from 'react';
import { shallow } from 'enzyme';
import CharacterList from './Character-list';

describe('Character List component', () => {
  it('renders Character List', () => {
    const wrapper = shallow(<CharacterList />);
    expect(wrapper).toMatchSnapshot();
  });
});
