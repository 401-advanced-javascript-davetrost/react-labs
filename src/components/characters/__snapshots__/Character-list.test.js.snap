// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character List component renders Character List 1`] = `
<div
  className="row"
>
  <Character
    characters={
      Array [
        Object {
          "img": "https://rickandmortyapi.com/api/character/avatar/327.jpeg",
          "name": "Slow Mobius",
          "species": "Humanoid",
        },
        Object {
          "img": "https://rickandmortyapi.com/api/character/avatar/370.jpeg",
          "name": "Two Guys with Handlebar Mustaches",
          "species": "Human",
        },
        Object {
          "img": "https://rickandmortyapi.com/api/character/avatar/155.jpeg",
          "name": "Harold",
          "species": "Cronenberg",
        },
        Object {
          "img": "https://rickandmortyapi.com/api/character/avatar/426.jpeg",
          "name": "Greasy Grandma",
        },
        Object {
          "img": "https://rickandmortyapi.com/api/character/avatar/487.jpeg",
          "name": "Visor Rick",
          "species": "Human",
        },
        Object {
          "img": "https://rickandmortyapi.com/api/character/avatar/265.jpeg",
          "name": "Pickle Rick",
          "species": "unknown, Pickle",
        },
        Object {
          "img": "https://rickandmortyapi.com/api/character/avatar/424.jpeg",
          "name": "Pizza-person",
          "species": "Humanoid, Pizza",
        },
      ]
    }
  />
</div>
`;
